static/spec/openapi.yaml

```diff
@@ -223,7 +223,7 @@ components:
         - partial
         - pending
         - running
-        - complete
+        - completed_at
```

Comment:
This references the status of export being `complete`, which is different from the time it was completed `completed_at`.